src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onFilter: (filterType: string) => void;
+  todos: Todo[];
+  filterBy: string;
+};
+
+export const Footer: React.FC<Props> = ({
+  onFilter,
+  todos,
+  filterBy,
+}) => {
+  const handleFilterChange = (filterType: string) => {
+    onFilter(filterType);
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos.length} items left`}
```

Comment:
u should show not todos.length, but number of todos u still have todo, it`s mean they are "active"